user/advanced-topics/bind-dirs.md

```diff
@@ -44,6 +44,14 @@ Inside the app qube.
 
 4. Save.
 
+> **Note**: If the directory you are trying to persist doesn't already exist in the app qube, you'll need to create the directory with its full path, under `/rw/bind-dirs`. In this case you would do:
+>
+>    ```
+>    sudo mkdir -p /rw/bind-dirs/var/lib/tor
+>    ```
+>
+> This case happens only when the template on which this app qube is based does have this directory.
+
```

Comment:
Should this just be its own step, i.e., number 5 in the list? (If the concern is that some users may not need to take any action, I think that's handled by the fact that this is phrased as a conditional, i.e., "*If* the directory you are trying to persist doesn't already exist....")

Reply:
Sure!